README.md

```diff
@@ -30,9 +30,11 @@ import smithy4s.codegen.Smithy4sCodegenPlugin
 val myModule = project
   .in(file("modules/my-module"))
   .enablePlugins(Smithy4sCodegenPlugin)
+  // version for smithy4s-core is sourced from Smithy4sCodegenPlugin
+  .settings(libraryDependencies += "com.disneystreaming.smithy4s" %%% "smithy4s-core" % smithy4sVersion.value)
```

Comment:
%%% won't work on builds without scalajs. In general, I think this approach is non-ideal (asking people to add the dependency themselves) but I don't have a better idea for now :/ hoping @Baccata has something better

Reply:
ah, I now see the comment in #34 saying it's already been discussed

Reply:
still, the recommendation should be `%%` I think

Reply:
Fair enough, I was trying to signal that it worked for ScalaJS as well, but maybe that is implied enough to change. And it is likely that ScalaJS folks will figure that out where a newcomer who isn't familiar with '%%%' might get confused

Reply:
💯